src/test/java/com/powsybl/openloadflow/sensi/ac/AcSensitivityAnalysisTest.java

```diff
@@ -362,13 +396,14 @@ void testBusVoltagePerTargetVTwt() {
             .map(busVoltage -> new BusVoltagePerTargetV(busVoltage, targetVoltage))
             .collect(Collectors.toList());
 
-        CompletionException e = assertThrows(CompletionException.class, () -> sensiProvider.run(network, VariantManagerConstants.INITIAL_VARIANT_ID, factorsProvider, Collections.emptyList(),
-            sensiParameters, LocalComputationManager.getDefault())
-            .join());
-
-        assertTrue(e.getCause() instanceof NotImplementedException);
+        SensitivityAnalysisResult result = sensiProvider.run(network, VariantManagerConstants.INITIAL_VARIANT_ID, factorsProvider, Collections.emptyList(),
+                sensiParameters, LocalComputationManager.getDefault()).join();
 
-        assertEquals("[T2wT] Bus target voltage on two windings transformer is not managed yet", e.getCause().getMessage());
+        assertEquals(4, result.getSensitivityValues().size());
+        assertEquals(0d, getValue(result, "T2wT", "BUS_1"), LoadFlowAssert.DELTA_V);
+        assertEquals(0.035205d,  getValue(result, "T2wT", "BUS_2"), LoadFlowAssert.DELTA_V);
+        assertEquals(1d,  getValue(result, "T2wT", "BUS_3"), LoadFlowAssert.DELTA_V);
```

Comment:
Just a thought: 
I had a doubt (not obvious when reading the code), that sensitivities are correctly "scaled back" from per-unit.
Finally it seems correct, but it looks like the logic is handled directly in the equation we solve for computing the sensitivity : I think at some point we should make more clear where is the limit between per-united values and "nominal" values (probably having a kind of conversion after having computed the sensitivities in per-unit).

Reply:
It is done in `calculateSensitivityValues` but it is not well done for BUS_VOLTAGE and I have some doubt about `sensi += Math.toRadians(factor.getEquationTerm().der(phi1Var));` line but no link with that PR :-)